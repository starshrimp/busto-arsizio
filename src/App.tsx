import * as React from 'react';
import { Jumbotron, Container } from 'react-bootstrap';

export const App: React.FC = () => (
  <Container>
    <Jumbotron fluid className="mt-5">
      <Container>
        <h1>SRM Page</h1>
        <p>My cool web page</p>
      </Container>
    </Jumbotron>
  </Container>
);

export default App;
