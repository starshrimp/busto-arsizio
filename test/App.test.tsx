import * as React from 'react';
import App from '../src/App';
import { shallow } from 'enzyme';

describe('App', () => {
  it('renders the app correctly', () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
