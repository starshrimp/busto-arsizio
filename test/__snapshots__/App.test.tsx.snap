// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the app correctly 1`] = `
<Container
  fluid={false}
>
  <Component />
  <Jumbotron
    className="mt-5"
    fluid={true}
  >
    <Container
      fluid={false}
    >
      <h1>
        Nweedia Dashboard
      </h1>
      <p>
        Management dashboard for the Nweedia infrastructure.
      </p>
    </Container>
  </Jumbotron>
</Container>
`;
